docs/design.md

```diff
@@ -104,3 +104,14 @@ several ways according to policy:
 
 Then like the usual LibOS pattern it will call the container's entry point as though the container
 was a process unto itself (rather than where it now sits like a library in the LibOS PAL process).
+
+## Local Attestation for FUSE Key Provisioning
+
+![EnclaveCC_LA.png](images/EnclaveCC_LA.png)
+
+In most scenarios, it is one-shot transactions used for single FUSE key provisioning. However, the LA protocol can easily be extended to provision multiple keys for more general purposes.
+To achieve this, it is necessary to add a key id to the second request message sent along G_a and app's report, with the ID being used to indicate which key is being requested.
+In addition to adding this ID, the generated nonce and derived AEK also need to be in the cache on both sides to keep the secure session alive which has been established in initial transaction.
```

Comment:
What `AEK` stands for?  Can you clarify "secure session" in the context of `ttrpc`.

Reply:
The `AEK` stands for algorithm encryption key, it is a derived key used for AEAD.
Regarding the secure session, it is tied to `AEK` and has shorter or equal lifetime than `ttrpc` connection. Thanks.

Reply:
Added to the acronym list: https://github.com/confidential-containers/documentation/wiki/Acronyms#aek